src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -3102,7 +3104,7 @@ TEST_P(conv_int8_quantize_u8_onednn, per_channel) {
         reorder("reorder_bfyx", "quantize", p.default_format, data_types::f32)
     );
 
-    tolerance = 1.0f;
+    tolerance = 1.f;
```

Comment:
why did you change this?

Reply:
Some code uses 1.f and the others uses 1.0f therefore I changed 1.0f to 1.f for code style consistency.